Meeting on 2024-11-19 (#168)
OpenRailAssociation/technical-committee: meetings/2024-11-19.md

> # TC Meeting 2024-11-05

Reviewer: ```suggestion
# TC Meeting 2024-11-19
```
(according to the name of the file?)

Present: Florian, Frederik, Tristram, Peter

## Notes

- Internal meeting about NGE moving the repository to OpenRail.
- OSRD: need to update the application for stage 2 (next week)
- LibLRS: No rush for stage 2
- DAC: Workshop on the model

>   - Frederik: Is open data is part of the OpenRail goals?

Reviewer: ```suggestion
  - Frederik: Is open data part of the OpenRail goals?
```

>   - Peter: No, but it's a close subject from open source

Reviewer: ```suggestion
  - Peter: No, but it is a close subject from open source
```

>   - Tristram: We have to be cautious, Railway Companies might be hesitent.

Reviewer: ```suggestion
  - Tristram: We have to be cautious, Railway Companies might be hesitant.
```
https://www.merriam-webster.com/thesaurus/hesitent

- Reminder: Fosdem deadline is 1st december
- Peter: "Security" requirements issue

> - Peter: Might have a new project from Infrabel: Library for "positioning mesurement vehicule".

Reviewer: ```suggestion
- Peter: Might have a new project from Infrabel: Library for "positioning measurement vehicule".
```
